src/content/docs/openai/ChatGPT/chatGPT-case-study-e0402.md

```diff
@@ -0,0 +1,174 @@
+
+- 1966: Joseph Weizenbaum, a German computer scientist and Professor at Massachusetts Institute of Technology, developed the program ELIZA inspired by Turing's paper. Eliza's purpose was to trick humans into believing they were chatting with another human and it's considered the first chat bot in human history. It didn't pass the Turing Test.
+
+- 1972: Kenneth Colby, a professor of psychiatry at Stanford University, develops Parry to study the human mind through computer models.
+
+- 1983: William Chamberlain and Thomas Etter create RACTER, a chat bot whose aim is to entertain people who interact with it.
+
+- 2000s: ALICE (Artificial Language Internet Computer Entity) was developed. Although it didn't pass the Turing Test, it became the foundation for most chatbots that exist today.
+
+- 2010: Apple launches their virtual assistant Siri.
+
+- 23rd March 2016: Microsoft releases Tay, a Twitter chat bot that was shut down shortly after because it learnt offensive language.
+
+- 6th November 2016: Amazon launches their virtual assistant Alexa.
+
+- 11th June 2018: OpenAI releases "Improving Language Understanding by Generative Pre-Training", a paper presenting the differences between natural language processing (NLP) models and GPT.
+
+- November 2019: OpenAI launches GPT-2 as a "direct scale-up" of GPT.
+
+- July 2020: GPT-3 is introduced in beta testing mode.
+
+- 29th June 2021: GitHub releases GitHub Copilot, developed in collaboration with OpenAI and based on GPT-3.
+
+- 5th August 2022: Meta launches Blenderbot 3, the company's most advanced AI chatbot to date.
+
+- 30th November 2022: OpenAI launches ChatGPT based on the GPT-3.5 model. It gained 1 million users within 5 days of its release, whereas other successful applications needed months to achieve the first million users. For instance, Netflix achieved its first million in 40 months, Facebook in 10 months, and Instagram in 2.5 months.
+
+- February 2023: Meta AI releases LLaMA (Large Language Model Meta AI).
+
+- February 6, 2023: Google announces Bard, a conversational chatbot AI powered by Meta AI's LLaMA.
+
+- March 14, 2023: OpenAI releases GPT-4; available via ChatGPT Plus and a commercial API.
+
+- March 30, 2023: Introduction of AutoGPT.
+
+- 2023: Release of BabyAGI and JARVIS/HuggingGPT.
+
+- Late 2023: OpenAI secures a $10 billion deal with Microsoft; expands ChatGPT to various industries and global markets.
+
+- December 2023: Reopening of ChatGPT Plus subscriptions and strategic partnership announcements.
+
+## Features
+
+ChatGPT has the ability to remember previous questions asked in a chat, enabling it to effectively interact with humans in a conversational manner. The model was trained on a vast amount of corpus data covering a wide range of topics and styles, making its potential uses diverse.
+
+When asked about what tasks it could assist with (screenshot #1), ChatGPT answered:
```

Comment:
Where is this screenshot?

Reply:
![chatgpt-4-screenshot](https://github.com/NoroffFEU/development-platforms/assets/100483961/79692acd-aa40-4612-add1-f1f83f1eecd7)